### PR TITLE
chore: linting and refactoring with scaffolding-cli package

### PR DESCRIPTION
**Pull request by estherthetesteramido**
#### 📲 What

Refactor with eslinting.

#### 🤔 Why
		
Because it was very pythonic.

I know the JSON has changed. But it's good practice to align with: https://google.github.io/styleguide/jsoncstyleguide.xml?showone=Property_Name_Format#Property_Name_Format
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [x] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [x] Meeting the Coding Standards?

### CODE REVIEW

File: packages/eslint-config/package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.17.0",
     "eslint-plugin-react-hooks": "^2.5.0",
-    "eslint-plugin-security": "^1.4.0"
+    "eslint-plugin-security": "^1.4.0",
+    "eslint-plugin-flowtype": "^5.1.0"
```
Comment (naren-bellala) [inline packages/eslint-config/package.json:26]: Why do we need this?

Comment (estherthetesteramido) [reply packages/eslint-config/package.json:26]: For linting, we need the peer dependencies as definied by airbnb.

File: packages/scaffolding-cli/src/domain/config/questions.ts [hunk 2]
```diff
@@ -1,171 +1,213 @@
...
+                    title: "API with Java",
+                    description: "api, java, springboot",
+                    value: "javaSpring",
                 },
                 {
-                    "title": "Selenium framework with .NET", "description": "automation, bddfy, xunit, webdriver, netcore, e2e, standalone", "value": "test_netcore_selenium"
+                    title: "Selenium framework with .NET",
+                    description:
+                        "automation, bddfy, xunit, webdriver, netcore, e2e, standalone",
+                    value: "test_netcore_selenium",
```
Comment (dnitsch) [inline packages/scaffolding-cli/src/domain/config/questions.ts:90]: do we want this as well to be camelCased

File: packages/scaffolding-cli/src/domain/config/questions.ts [hunk 3]
```diff
@@ -1,171 +1,213 @@
...
                 },
                 {
-                    "title": "TestCafe framework with Typescript", "description": "in-browser, automation, node.js, javascript, bdd, cross browser", "value": "test_js_testcafe"
+                    title: "TestCafe framework with Typescript",
+                    description:
+                        "in-browser, automation, node.js, javascript, bdd, cross browser",
+                    value: "test_js_testcafe",
```
Comment (dnitsch) [inline packages/scaffolding-cli/src/domain/config/questions.ts:96]: ibid

File: packages/scaffolding-cli/src/domain/config/questions.ts [hunk 4]
```diff
@@ -1,171 +1,213 @@
...
-            "choices": [
+            type: "select",
+            name: "deployment",
+            message: "Select Pipeline Tool",
+            choices: [
                 {
-                    "title": "AzureDevOps", "description": "Azure Devops/VSTS/TFS", "value": "azdevops"
+                    title: "AzureDevOps",
+                    description: "Azure Devops/VSTS/TFS",
+                    value: "azdevops",
```
Comment (dnitsch) [inline packages/scaffolding-cli/src/domain/config/questions.ts:123]: same as above?

File: packages/scaffolding-cli/src/domain/config/questions.ts [hunk 5]
```diff
@@ -1,171 +1,213 @@
...
-            "choices": [
+            type: "select",
+            name: "terraformBackendStorage",
+            message: "Select Terraform remote state storage service?",
+            choices: [
                 {
-                    "title": "Azure", "description": "Azure Blob", "value": "azure_blob"
+                    title: "Azure",
+                    description: "Azure Blob",
+                    value: "azure_blob",
```
Comment (dnitsch) [inline packages/scaffolding-cli/src/domain/config/questions.ts:158]: these as well?

File: packages/scaffolding-cli/src/domain/config/sample-infra.bootstrap-config.json [hunk 6]
```diff
@@ -1,19 +1,19 @@
...
-  "business_project": "replace_business_project",
-  "business_domain": "replace_business_domain",
-  "business_component": "replace_business_component",
-  "source_control_repo_type": "replace_source_control_repo_type",
-  "source_control_repo_name": "replace_source_control_repo_name",
-  "source_control_repo_url": "replace_source_control_repo_url",
-  "terraform_backend_storage": "replace_terraform_backend_storage",
-  "terraform_backend_storage_rg": "replace_terraform_backend_storage_rg",
-  "terraform_backend_storage_container": "replace_terraform_backend_storage_container"
+  "advancedConfig": true,
```
Comment (dnitsch) [inline packages/scaffolding-cli/src/domain/config/sample-infra.bootstrap-config.json:6]: this can be removed `advancedConfig`

**Comment by estherthetesteramido** [comment]
Solves: https://github.com/amido/stacks-webapp-template/pull/214, and https://github.com/amido/stacks-webapp-template/pull/216. Required refactoring with verison updates/

**Pull request merged**